gravitee-plugin-core/pom.xml

```diff
@@ -43,10 +43,22 @@
             <artifactId>gravitee-common</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.reactivex.rxjava3</groupId>
+            <artifactId>rxjava</artifactId>
+            <version>3.1.6</version>
```

Comment:
You should refer to a specific version here, gravitee-bom should already declare it.